main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Main.java

```diff
@@ -154,4 +161,13 @@ private static void exportReport(final NoticeContainer noticeContainer, final Ar
       logger.atSevere().withCause(e).log("Cannot store report files");
     }
   }
+
+  /**
+   * Prints help menu
+   *
+   * @param jCommander the {@code JCommander} used to parse CLI arguments
+   */
+  private static void printHelp(JCommander jCommander) {
+    jCommander.usage();
```

Comment:
Please inline a call to this method and remove `printHelp` since it is too tiny and does not add readability.